<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganizationsWindow</class>
 <widget class="QMainWindow" name="OrganizationsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="header_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <widget class="QLabel" name="org_name_lable">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>261</width>
         <height>44</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="organization_name_text">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>10</y>
         <width>214</width>
         <height>44</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Organization:</string>
       </property>
      </widget>
      <widget class="QLabel" name="welcome_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Welcome </string>
       </property>
      </widget>
      <widget class="QLabel" name="user_name_text">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>name</string>
       </property>
      </widget>
      <widget class="QLabel" name="username_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>your username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="user_username_text">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>username</string>
       </property>
      </widget>
      <widget class="QLabel" name="user_role_text">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Not defiened</string>
       </property>
      </widget>
      <widget class="QLabel" name="role_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>your role:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="body_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="Persons">
          <attribute name="title">
           <string>Persons</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTreeWidget" name="org_persons_tree_widget">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="headerShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Username</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Org ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>is Admin</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="search_org_per_by_text">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Search by:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="search_org_per_by_name_radio">
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="search_org_per_by_username_radio">
                <property name="text">
                 <string>Username</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="search_org_per_by_org_ID_radio">
                <property name="text">
                 <string>Org ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="org_users_search_line_edit">
                <property name="placeholderText">
                 <string>Search by Name, Username or ID ...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="is_org_member_checkbox">
                <property name="text">
                 <string>Members</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="is_org_admin_checkbox">
                <property name="text">
                 <string>Admins</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="Persons_widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="org_page_add_person_BTN">
                <property name="text">
                 <string>add new person</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="org_page_edit_person_BTN">
                <property name="text">
                 <string>change role</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="org_page_remove_person_BTN">
                <property name="text">
                 <string>remove person</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Teams">
          <attribute name="title">
           <string>Teams</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTreeWidget" name="teams_tree_widget">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>Team name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Team ID</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="add_team_BTN">
                <property name="text">
                 <string>add a team</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="edit_team_name_BTN">
                <property name="text">
                 <string>edit team name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove_team_BTN">
                <property name="text">
                 <string>remove team</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="search_teams_line_edit">
                <property name="placeholderText">
                 <string>search ...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Projects">
          <attribute name="title">
           <string>Projects</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTreeWidget" name="projects_tree_widget">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>Project Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Project Admin</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="add_project_BTN">
                <property name="text">
                 <string>add new project</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="edit_project_BTN">
                <property name="text">
                 <string>edit project</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove_project_BTN">
                <property name="text">
                 <string>remove project</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLineEdit" name="search_projects_line_edit">
                <property name="placeholderText">
                 <string>search ...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
